src/openrct2/EditorObjectSelectionSession.cpp

```diff
@@ -64,26 +64,6 @@ static constexpr ResultWithMessage ObjectSelectionError(bool isMasterObject, Str
  */
 static void setup_track_manager_objects()
 {
-    int32_t numObjects = static_cast<int32_t>(object_repository_get_items_count());
-    const ObjectRepositoryItem* items = object_repository_get_items();
-    for (int32_t i = 0; i < numObjects; i++)
-    {
-        uint8_t* selectionFlags = &_objectSelectionFlags[i];
-        const ObjectRepositoryItem* item = &items[i];
-        if (item->Type == ObjectType::Ride)
-        {
-            *selectionFlags |= ObjectSelectionFlags::Flag6;
-
-            for (auto rideType : item->RideInfo.RideType)
-            {
-                if (GetRideTypeDescriptor(rideType).HasFlag(RIDE_TYPE_FLAG_HAS_TRACK))
-                {
-                    *selectionFlags &= ~ObjectSelectionFlags::Flag6;
-                    break;
-                }
-            }
-        }
-    }
```

Comment:
Should I remove this method since I am removing all of its contents? The "rct2: ..." comment led me to think these methods may be tied to something else.

Reply:
I added this method back to fully disable shops (which I found my changes before had not done). So the question of whether to remove this method no longer needs an answer.